data/language/dutch.txt

```diff
@@ -3176,12 +3176,12 @@ STR_3174    :Een ander object is afhankelijk van dit object
 STR_3175    :Dit object is altijd benodigd
 STR_3176    :Kan object niet selecteren
 STR_3177    :Kan object niet deselecteren
-STR_3178    :Er moet minstens één voetpad worden geselecteerd
-STR_3179    :Er moet minstens één attractie worden geselecteerd
+STR_3178    :Er moet minimaal één voetpad worden geselecteerd
+STR_3179    :Er moet minimaal één attractie worden geselecteerd
```

Comment:
Volgens mij is minimaal eigenlijk een bijvoeglijk naamwoord. Ten minste of minstens is hier dus beter.